src/WebJobs.Script/BindingExtensions/ExtensionsManager.cs

```diff
@@ -185,15 +185,6 @@ internal virtual Task ProcessExtensionsProject(string projectFolder)
 
         private void SetupProcessEnvironment(ProcessStartInfo startInfo)
         {
-            foreach (DictionaryEntry environmentVariable in Environment.GetEnvironmentVariables(EnvironmentVariableTarget.Process))
-            {
-                if (!TryAdd(startInfo.Environment, environmentVariable.Key?.ToString(), environmentVariable.Value?.ToString()))
-                {
-                    _logger.LogWarning($"Unable to set environment variable '{environmentVariable.Key}' during extension installation. "
-                        + "A key with the same name already exists");
-                }
-            }
```

Comment:
Any idea why this was there?

Reply:
No. I recall some reason, but don't remember the details. I don't think those were valid.